### PR TITLE
[robotraconteur] Update to version 1.1.1

### PR DESCRIPTION
**Pull request by johnwason**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- [x] SHA512s are updated for each updated download.
- [ ] The "supports" clause reflects platforms that may be fixed by this new version.
- [ ] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [ ] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/robotraconteur/portfile.cmake [hunk 1]
```diff
@@ -5,8 +5,8 @@ endif()
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
 	REPO robotraconteur/robotraconteur
-	REF v1.0.0
-	SHA512 c21dd0af579272c565dd66ca935aababfa3742db524fae66fe82c929561680608e5759ce954f31a8bbcb4ffb7c4e5314f2050b513ccddb8fb49a005c6cfa6d74
+	REF v${VERSION}
```
Comment (FrankXie05) [inline ports/robotraconteur/portfile.cmake:8]:
```suggestion
	REF "v${VERSION}"
```

**Review by FrankXie05 — CHANGES_REQUESTED** [review]

File: ports/robotraconteur/portfile.cmake (hunk 1 above)
Comment (johnwason) [reply ports/robotraconteur/portfile.cmake]: Done

**Review by FrankXie05 — APPROVED** [review]

**Comment by BillyONeal** [comment]
I want this to land after merged with https://github.com/microsoft/vcpkg/pull/37466 after it lands since that PR turns on testing for this in CI

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**